src/vs/workbench/services/files/test/node/fileService.test.ts

```diff
@@ -136,6 +136,33 @@ suite('FileService', () => {
 		}, error => onError(error, done));
 	});
 
```

Comment:
@tsalinger I do not really understand why you added new tests in the fileService, when this feature does not touch anything inside the fileService as far as I understand
@bpasero can you please review if these tests are needed in the `fileService`

Reply:
These are exploratory tests that I wrote to test whether I can implement this feature the way I did.